app/src/main/java/de/danoeh/antennapod/ui/home/HomeFragment.java

```diff
@@ -92,7 +92,7 @@ private void populateSectionList() {
         viewBinding.homeContainer.removeAllViews();
 
         SharedPreferences prefs = getContext().getSharedPreferences(HomeFragment.PREF_NAME, Context.MODE_PRIVATE);
-        if (Build.VERSION.SDK_INT >= 33 && ContextCompat.checkSelfPermission(getContext(),
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && ContextCompat.checkSelfPermission(getContext(),
```

Comment:
Please keep the API version number. The desert names are not used anywhere else (eg Gradle, Android documentation, etc). That makes them harder to read than just the number itself

Reply:
Fair enough